core/sail/inferencer/src/main/java/org/eclipse/rdf4j/sail/inferencer/fc/SchemaCachingRDFSInferencerConnection.java

```diff
@@ -767,17 +767,13 @@ public void rollback()
 	@Override
 	public void begin()
 			throws SailException {
-		this.begin(null);
+		this.begin(sail.getDefaultIsolationLevel());
 	}
 
 	@Override
 	public void begin(IsolationLevel level)
 			throws SailException {
 
-		if (level == null) {
```

Comment:
If this check is being removed, then there should be a new null-guard on this parameter, given it is a public API method that previously supported null as a valid parameter value (even if `IsolationLevels.getCompatibleIsolationLevel` is currently null-safe)

Reply:
fixed